specification/esql/query/QueryRequest.ts

```diff
@@ -67,10 +74,29 @@ export interface Request extends RequestBase {
      * @doc_id esql-query-params
      */
     params?: Array<FieldValue>
+    /**
+     * If provided and `true` the response will include an extra `profile` object
+     * with information on how the query was executed. This information is for human debugging
+     * and its format can change at any time but it can give some insight into the performance
+     * of each part of the query.
+     */
     profile?: boolean
     /**
      * The ES|QL query API accepts an ES|QL query string in the query parameter, runs it, and returns the results.
      */
     query: string
+    /**
+     * Tables to use with the LOOKUP operation.
```

Comment:
We should probably document the dictionary keys a little bit more in detail. I assume the outer key is the table name and the inner one the column name?

Reply:
Done.